streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -101,6 +101,7 @@
     public void before() {
         props = new Properties();
         props.put(StreamsConfig.APPLICATION_ID_CONFIG, "appId");
+        props.put(StreamsConfig.CLIENT_ID_CONFIG, "appId");
```

Comment:
Nit: why reuse `appId` -- if there is a bug, and we set `APPLICATION_ID_CONFIG` by mistake, it would not be detected. Maybe better to set a different id.

Reply:
good call.